lms/djangoapps/edxnotes/views.py

```diff
@@ -66,6 +68,32 @@ def edxnotes(request, course_id):
     return render_to_response("edxnotes/edxnotes.html", context)
 
 
+@require_GET
+@login_required
+def list_notes(request, course_id):
```

Comment:
Both views are not same, first one is used to render a mako template and the second one is used for pagination.